app/wama.web/Controllers/ReportToolController.cs

```diff
@@ -143,12 +145,25 @@ public IActionResult CheckInReports(ReportToolFilterViewModel filter)
             return View($"{Constants.ADMIN_CONSOLE_REPORT_TOOL_DIRECTORY}/CheckIns.cshtml", result);
         }
 
-        public async Task<IActionResult> Users()
+        [HttpGet]
+        public IActionResult Users()
+        {
+            return View($"{Constants.ADMIN_CONSOLE_REPORT_TOOL_DIRECTORY}/Users.cshtml");
```

Comment:
If this is the file that displays the filter, then give the file some meaningful name